packages/react-next/src/components/Keytip/examples/Keytips.Basic.Example.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { keytipMap } from 'office-ui-fabric-react/lib/components/Keytip/examples/KeytipSetup';
+import { Checkbox } from '@fluentui/react-next/lib/Checkbox';
+import { ComboBox } from '@fluentui/react-next/lib/ComboBox';
+import { Dropdown } from '@fluentui/react-next/lib/Dropdown';
+import { Link } from '@fluentui/react-next/lib/Link';
+import { SpinButton, ISpinButtonStyles } from '@fluentui/react-next/lib/SpinButton';
+import { Toggle } from '@fluentui/react-next/lib/Toggle';
+import { Pivot, PivotItem } from '@fluentui/react-next/lib/Pivot';
+import { IStackTokens, Stack } from '@fluentui/react-next/lib/Stack';
+import { useKeytipData } from '@fluentui/react-next/lib/KeytipData';
+
+const pivotItemStyle: React.CSSProperties = { width: 500, paddingTop: 20 };
+const stackTokens: IStackTokens = { childrenGap: 20 };
+const spinButtonStyles: Partial<ISpinButtonStyles> = { root: { maxWidth: 200 } };
+
+const sampleOptions = [
+  { key: 'A', text: 'Option 1' },
+  { key: 'B', text: 'Option 2' },
+  { key: 'C', text: 'Option 3' },
+];
+
+export const KeytipsBasicExample: React.FunctionComponent = () => {
+  const checkboxKeytipData = useKeytipData({ keytipProps: keytipMap.CheckboxKeytip });
+  const linkKeytipData = useKeytipData({ keytipProps: keytipMap.LinkKeytip });
+  const toggleKeytipData = useKeytipData({ keytipProps: keytipMap.ToggleKeytip });
+
+  return (
+    <div>
+      <p>
+        For Pivots, keytips will first show for each of the pivots. After selecting a pivot, the Keytips for its content
+        are shown.
+      </p>
+      <Pivot>
+        <PivotItem headerText="Pivot 1" keytipProps={keytipMap.Pivot1Keytip} style={pivotItemStyle}>
+          <Stack tokens={stackTokens}>
+            <SpinButton label="Spin Button" keytipProps={keytipMap.SpinButtonKeytip} styles={spinButtonStyles} />
+            <Toggle onText="Yes" offText="No" keytipData={toggleKeytipData} />
+            <span>
+              Go to{' '}
+              <Link keytipData={linkKeytipData} href="http://www.bing.com" target="_blank">
```

Comment:
Before:
```
<Link keytipProps={keytipMap.LinkKeytip} href="http://www.bing.com" target="_blank">
```

After:
```
const linkKeytipData = useKeytipData({ keytipProps: keytipMap.LinkKeytip });
<Link keytipData={linkKeytipData} href="http://www.bing.com" target="_blank">
```

Reply:
Instead why not this?

```tsx
<Keytip {...keytipProps}>
  <Link href="..." target="...">...</Link>
</Keytip>
```

Reply:
Updated the code:

Before:
```
<Link keytipProps={keytipMap.LinkKeytip} href="http://www.bing.com" target="_blank">
```

After:
```
const linkRef = useKeytipRef({ keytipProps: keytipMap.LinkKeytip });
<Link ref={linkRef} href="http://www.bing.com" target="_blank">
```